safehttp/plugins/csp/csp.go

```diff
@@ -233,17 +241,34 @@ func (it Interceptor) Before(w safehttp.ResponseWriter, r *safehttp.IncomingRequ
 	for _, p := range it.ReportOnly {
 		reportCSPs = append(reportCSPs, p.Serialize(nonce))
 	}
+	// X-Frame-Options works only in enforce mode.
+	var XFO = []string{getXFO(&it.Enforce)}
 
 	h := w.Header()
 	setCSP := h.Claim("Content-Security-Policy")
 	setCSPReportOnly := h.Claim("Content-Security-Policy-Report-Only")
+	setXFO := h.Claim("X-Frame-Options")
 
 	setCSP(CSPs)
 	setCSPReportOnly(reportCSPs)
+	setXFO(XFO)
 
 	return safehttp.NotWritten()
 }
 
+// getXFO returns X-Frame-Options header value that reflects configuration of the FramingPolicy.
+// If no FramingPolicy exists, defaults to SAMEORIGIN.
+func getXFO(p *[]Policy) string {
```

Comment:
It returns the first XFrameOptions, right? Consider naming it something along the lines of `firstXFrameOptions`

"get" is usually treated as a filler word in Go function names as it doesn't add anything to the meaning of it. It's almost always better to drop it altogether or replace it with a more specific word (e.g. `fetch` if it involves an RPC).